app/build.gradle

```diff
@@ -15,7 +15,7 @@ android {
         minSdkVersion 14
         targetSdkVersion 26
         versionCode 1
-        versionName "2.0"
+        versionName "0.1"
```

Comment:
This is lower than the current store version

Reply:
I did not see any tags here so I did not expect that the app is already published and starting with a version number 2 look a bit strange. But for sure its finally not really relevant.

Reply:
It's actually at 0.2 - https://play.google.com/store/apps/details?id=org.piwigo
As it's a complete rewrite it seemed easier just to call it 2.0 (the iOS app was released as version 2.x as well)

Reply:
That's a point. If you consider the app here as an update of the existing IOS-port than version 2.0 totally makes sense!